intro-HPC/ch_jobscript_examples.tex

```diff
@@ -7,25 +7,36 @@ \section{Single-core job}
 
 \examplecode{bash}{single_core.sh}
 
+\begin{enumerate}
+    \item We set \lstinline|#PBS| headers, see \autoref{ch:torque-options} for a list of these options
+    \item We load a module, see \autoref{sec:modules}
+    \item We stage the data in
```

Comment:
mention `input.txt`, explain what stage in means (file is copied into the "working" directory), refer to chapter/section about I/O?